pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/BundleSplitterTask.java

```diff
@@ -82,6 +82,11 @@ public BundleSplitterTask(final PulsarService pulsar) {
                     totalMessageRate = longTermData.totalMsgRate();
                     totalMessageThroughput = longTermData.totalMsgThroughput();
                 }
+                // if there are just one topic, avoid to infinitely split
+                if (stats.topics == 1) {
```

Comment:
Could you please help add a test for covering the new change?

Reply:
addded